spec/factories/mapping_factory.rb

```diff
@@ -9,8 +9,13 @@
     kind { 'view' }
     association :ontology
 
-    after(:build) do |mapping|
-      mapping.locid = "#{mapping.ontology.locid}//#{mapping.name}"
+    after(:create) do |mapping|
+      LocId.where(
+                    locid: "#{mapping.ontology.locid}//#{mapping.name}",
+                  ).first_or_create!(
+                  assorted_object_id: mapping.id,
+                  assorted_object_type: 'Mapping',
```

Comment:
This may not be a relevant request, but I'd rather use `Mapping` instead of `'Mapping'` because then we should get a better error if there is something wrong with class loading or typos.